backend/wayland/backend.c

```diff
@@ -18,6 +18,11 @@
 #include "util/signal.h"
 #include "xdg-decoration-unstable-v1-client-protocol.h"
 #include "xdg-shell-client-protocol.h"
+#include "tablet-unstable-v2-client-protocol.h"
+
+struct wlr_wl_tablet_seat *wlr_wl_add_tablet_seat(
```

Comment:
1. This isn't a public function, so it shouldn't have the `wlr_` prefix. `add_wl_tablet_seat` would be a better name.
2. This should be declared in the private header.